changelog.d/20241120_161839_danfuchs_event_manager_test_failures.md

```diff
@@ -0,0 +1,7 @@
+### Backwards-incompatible changes
+
+- `logger` value to `safir.metrics.KafkaMetricsConfiguration.make_manager` is passed through to the `PydanticSchemaManager` instance that it creates.
```

Comment:
I don't think this is sufficiently backwards-incompatible to warrant the major version bump. I'd toss it under new features. In general, I've followed a policy that changes to logging don't count as backward-incompatible changes.